is-wds-admin.php

```diff
@@ -247,11 +247,13 @@ function wds_is_admin() {
 
 /**
  * Checks if the current user is wds_admin and has special capabilities.
- * @return boolean
+ *
+ * @since  1.0.0
+ * @return boolean Is the user WDS Admin?
  */
 function is_wds_admin() {
 	return wds_is_admin()->is_wds_admin();
 }
 
 // Kick it off.
-add_action( 'admin_init', array( wds_is_admin(), 'add_cap_if_not_exists' ) );
```

Comment:
Without having this class on a hook of some sort, we'll run the risk of firing this class everywhere. Is that something we really want to do? Thought this was for Admin only?